wordpress-mobile/WordPress-iOS
# Jetpack Focus: Implement design for the new users overlay

Part of #19453

## Description
This PR implements the overlay for new users. The design for this overlay is different from the existing overlays, which required adding a way to customize the generic overlay.

## ⚠️ Disclaimer

The focus of this PR is only on the design of the overlay.
The menu card and showing the overlay on login will be implemented in a future PR.

### Screenshots

| Light Mode | Dark Mode |
| - | - |
|![light](https://user-images.githubusercontent.com/25306722/211454959-be17e790-bc20-4bce-9895-7dde2ce1ee06.png)|![dark](https://user-images.githubusercontent.com/25306722/211454964-2e7418e2-d4b9-4871-84ad-098c1f4b17c6.png)|

## Pre-testing Instructions
- To disable the learn more button:
    - Go to `JetpackFullscreenOverlayGeneralViewModel.swift` line 183
    - Replace this line with `return nil`
    - This won't be needed in production since the learn more button will be disabled remotely by then
- To always show the overlay even if it was shown before:
    - Go to `JetpackFeaturesRemovalCoordinator.swift` line 132
    - Change the default value to `true`

## Testing Instructions

1. Open the app
2. Open the debug menu and enable "Jetpack Features Removal Phase for New Users"
3. Navigate to My Site
4. Close and re-open the app
5. The overlay should be displayed
6. Make sure that the overlay matches the design

## Regression Notes
1. Potential unintended areas of impact
The design of other overlays.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
